sys/libraries/mailer.php

```diff
@@ -0,0 +1,26 @@
+<?php
+class mailer extends Library {
+
+	function __construct() {
+		$olddir = getcwd();
+		chdir(dirname(__FILE__));
+		require("PHPMailer/class.phpmailer.php");
+		//require('PHPMailer/PHPMailerAutoload.php');
```

Comment:
Why didn't you use autoloader?

And, We should avoid adding useless commented codes (remember SSAD! :P)